torch_xla/csrc/ops/ops.cpp

```diff
@@ -151,6 +151,18 @@ NodePtr NllLossBackwardOp(const Value& logits, const Value& labels) {
                             std::move(lower_fn));
 }
 
+NodePtr NotSupportedOp(c10::Symbol node_symbol) {
+  auto lower_fn = [](const ir::Node& node,
+                     ir::LoweringContext* loctx) -> ir::XlaOpVector {
+    XLA_ERROR() << "Node not supported: " << node.ToString();
```

Comment:
XLA_ERROR leads to a noreturn function attribute.

Reply:
Done.